addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-cidr-service.js

```diff
@@ -54,6 +54,14 @@ class EnvironmentScCidrService extends Service {
     const erroneousInputs = [];
     const ipv6Format = [];
     const protPortCombos = {};
+    if (
+      _.some(
+        updateRequest,
+        rule => rule.fromPort < 0 || rule.fromPort > 65535 || rule.toPort < 0 || rule.toPort > 65535,
```

Comment:
Can these ranges be added to JSON schema, so that we don't have to explicitly check the range?

Reply:
Done.